File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.cpp
#include <string>
#include <vector>
#include <cctype>

using namespace std;

string solution(string s) {
    string answer = "";
    int idx = 0;
    for(auto x: s) {
        if (x == ' ') {
            idx = 0;
            answer += x;
            continue;
        }
        
        if (idx % 2 == 0) {
            answer += toupper(x);
        } else {
            answer += tolower(x);
        }
        idx++;
    }
    return answer;
}

File: 프로그래머스/1/12930. 이상한 문자 만들기/이상한 문자 만들기.py
def solution(s):
    changes = []
    for word in s.split(" "):
        tmp = ""
        for idx, w in enumerate(word):
            if idx % 2 == 0:
                tmp += w.upper()
            else:
                tmp += w.lower()
        changes.append(tmp)
    
    answer = " ".join(changes)
    return answer